docs/user_guide/emummc/making_emummc.md

```diff
@@ -5,7 +5,7 @@
 ### Making the emuMMC
 
 !!!warning "Before you start"
-    Before you start, boot your switch normally, and delete all the wifi networks. You can add them back to your sysnand after completing this guide
+    If you don't plan on using the Internet on your emuMMC, it is recommended to boot your switch normally and delete all the wifi networks. You can add them back to your sysnand after completing this guide.
```

Comment:
small i in internet, also debatable if this step is really needed because the hosts file for emummc exists anyway

Reply:
internet done
Pending the outcome of the debate